admin/aws.py

```diff
@@ -126,11 +126,23 @@ def perform_copy_s3_keys(dispatcher, intent):
     """
     s3 = boto.connect_s3()
     destination_bucket = s3.get_bucket(intent.destination_bucket)
+    source_bucket = s3.get_bucket(intent.source_bucket)
     for key in intent.keys:
-        destination_bucket.copy_key(
-            new_key_name=intent.destination_prefix + key,
-            src_bucket_name=intent.source_bucket,
-            src_key_name=intent.source_prefix + key)
+        if '/_images/' in key and key.endswith('.svg'):
```

Comment:
Check for `/_images/` seems fragile? And I can't imagine we'd ever upload something to docs that ends with `.svg` that isn't a SVG.